vault/resource_rabbitmq_secret_backend_role.go

```diff
@@ -168,12 +168,19 @@ func rabbitMQSecretBackendRoleRead(d *schema.ResourceData, meta interface{}) err
 	d.Set("backend", strings.Join(pathPieces[:len(pathPieces)-2], "/"))
 	d.Set("name", pathPieces[len(pathPieces)-1])
 
-	if err := d.Set("vhost", flattenRabbitMQSecretBackendRoleVhost(secret.Data["vhosts"].(map[string]interface{}))); err != nil {
-		return fmt.Errorf("Error setting vhosts in state: %w", err)
+	vhosts := secret.Data["vhosts"]
+	vhostTopics := secret.Data["vhost_topics"]
```

Comment:
nit: related variables should usually be proximal to their usage.

Reply:
Removed in bdf259f